### PR TITLE
4659 documentation for vuetify element tree view

### PR DESCRIPTION
**Pull request by OlgaPoddubnaia**

### CODE REVIEW

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -4247,107 +4247,203 @@ v-parallax element has following methods:
...
-`content` | used to set _content_ in the _root_ of TreeView
-`full` | used to set class that used as a sign of _full_ checked checkbox
-`part` | used to set class that used as a sign of _partly_ checked checkbox
-`not` | used to set class that used as a sign of _not_ checked checkbox
+| Locator in `JDITreeView` | Description                                                        
+--------------------------|--------------------------------------------------------------------
+ `core`                   | used to set the _core_ of the TreeView                             
+ `full`                   | used to set class that used as a sign of _full_ checked checkbox   
+ `part`                   | used to set class that used as a sign of _partly_ checked checkbox 
+ `not`                    | used to set class that used as a sign of _not_ checked checkbox
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:4263]: why did you choose only 4? just as an example? :)

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -4247,107 +4247,203 @@ v-parallax element has following methods:
...
+ **allNodes()**         | Get TreeView check list               | WebList            
+ **node(String value)** | Get TreeView node string value        | TreeViewNode       
+ **node(int index)**    | Get TreeView item with required index | TreeViewNode       
+ **treeViewNodes()**    | Get TreeView list of nodes            | List<TreeViewNode> 
+ **isHoverable()**      | Get if TreeView is hoverable          | boolean            
+ **fullSize()**         | Get TreeView size                     | int                
+ **expandAllNodes()**   | Expand all nodes in TreeView          | void               
+ **closeAllNodes()**    | Close all nodes in TreeView           | void               
+
+In addition, **TreeView** extends implements ISetup, IsLoading, ISelector, IsDense.
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:4381]: extends? :)

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 1 above)
Comment (OlgaPoddubnaia) [reply source/includes/documentation/_vue_js_elements.md:4269]: discussed it with Vladimir and took them back

File: source/includes/documentation/_vue_js_elements.md (hunk 2 above)
Comment (OlgaPoddubnaia) [reply source/includes/documentation/_vue_js_elements.md]: deleted extends

**Review by MayaElf — APPROVED** [review]

**Pull request merged**